### PR TITLE
[General][Minor]Update Code Onwers

### PR DESCRIPTION
**Pull request by MyronFanQiu**
remove @zikalino from code owner

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -22,10 +22,11 @@
 /src/azure-cli/azure/cli/command_modules/dls/ @lewu-msft
 /src/azure-cli/azure/cli/command_modules/eventgrid/ @kalyanaj
 /src/azure-cli/azure/cli/command_modules/extension/ @zikalino
-/src/azure-cli/azure/cli/command_modules/keyvault/ @jiasli
+/src/azure-cli/azure/cli/command_modules/keyvault/ @bim-msft @jiasli
```
Comment (fengzhou-msft) [inline .github/CODEOWNERS]: Please also add me for keyvault.

Comment (MyronFanQiu) [reply .github/CODEOWNERS]: ok. @jiasli  Should I remove you?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,10 +4,10 @@
 *help.yaml @sptramer @Juliehzl
 *help.yml  @sptramer @Juliehzl
 
-/tools/ @zikalino
-/scripts/ @zikalino
-/src/azure-cli-testsdk/ @zikalino
-/src/azure-cli-core/ @jiasli
+/tools/ @haroldrandom @fengzhou-msft
+/scripts/ @haroldrandom
+/src/azure-cli-testsdk/ @bim-msft @MyronFanQiu
```
Comment (haroldrandom) [inline .github/CODEOWNERS]: Add me /src/azure-cli-testsdk/

**Comment by jsntcy** [comment]
Could we add at least two code owners for each module in case the only code owner is not available? @

**Comment by fengzhou-msft** [comment]
Added myself to /scripts.

**Pull request merged**